src/test/java/plugins/StageViewTest.java

```diff
@@ -21,7 +21,7 @@
  * needs adaption for future blue ocean
  * @author Boris Dippolter
  */
-@WithPlugins("workflow-aggregator")
+@WithPlugins({"pipeline-stage-view", "workflow-durable-task-step", "workflow-basic-steps"})
```

Comment:
Seems that this picks up `workflow-job` via `pipeline-stage-view` (likely a design flaw in the latter); what about `workflow-cps` though?

Reply:
I've had a bit of a look through and have no idea from a quick look of trawling dependencies on https://plugins.jenkins.io/workflow-cps/#dependencies

/

https://plugins.jenkins.io/pipeline-stage-view/#dependencies

Reply:
OK…but it passes somehow nonetheless?

Reply:
Yup passed locally and on CI